src/PHPUnit/Controller/AbstractHttpControllerTestCase.php

```diff
@@ -744,7 +744,7 @@ private function queryContentContainsAssertion($path, $match, $useXpath = false)
     {
         $result = $this->query($path, $useXpath);
 
-        if ($result->count() === 0) {
+        if (! $result->count()) {
```

Comment:
https://github.com/symfony/dom-crawler/blob/6.1/Crawler.php#L1020 returns an `int`, so I think this is OK-ish, although implicitly casting to a boolean is more of a stylistic choice (and less precise), so IMO this change shouldn't be done.

Let's say, for example, that `$result->count()` changes from `int` to `int|null` in the future: is `null` still supposed to pass this condition?